Python/pdf_reorder/generate_pdf.py

```diff
@@ -0,0 +1,39 @@
+
+## This script can pe used for testing the script pdf_reorder.py
+## with a random pdf with pages in wrong order 
+## I have written this script so that
+## the pages on generated pdf will have a page with numbers
+## ofcourse in wrong order ;)
+
+#shuffle is used to shuffle a list
+from random import shuffle 
+from fpdf import FPDF
+
+# save FPDF() class into a 
+# variable pdf 

```

Comment:
Comments are there to make the Code more readable, but if Comments are forcefully inserted, the readability goes on a downhill. Remove common comments since the Code is readable enough and instead focus on explaining the core logic part

Reply:
Okay